src/components/App/App.js

```diff
@@ -1,80 +1,34 @@
-import React, {useState, useEffect} from 'react';
-import ItemList from '../ItemList/ItemList';
-import InputItem from '../InputItem/InputItem';
-import Footer from '../Footer/Footer';
-import styles from './App.module.css';
-
-
-const App = () => {
-	const initialState = {
-		items: [
-			{
-				value: 'Покормить кота',
-				isDone: true,
-				id: 1
-			},
-			{
-				value: 'Сходить на работу',
-				isDone: false,
-				id: 2
-
-			},
-			{
-				value: 'Полить цветы',
-				isDone: true,
-				id: 3
-			}
-		],
-
-		count: 3,
-	};
-
-const [items, setItems] = useState(initialState.items);
-const [count, setCount] = useState(initialState.count);
-
-useEffect(() => {console.log('componentDidMount')}, []);
-useEffect(() => {console.log('componentDidUpdate')});	
+import React from 'react';
+import {
+    BrowserRouter as Router,
+    Route,
+    Link,
+} from 'react-router-dom';
+import Todo from '../Todo/Todo';
+import About from '../About/About';
+import Contacts from '../Contacts/Contacts';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
 
-const onClickDone = id => {
-	const newItemList = items.map(item => {
-		const newItem = {...item};
-		if (item.id === id) {
-			newItem.isDone = !item.isDone
-		}
-
-		return newItem;
-	});
-
-	setItems(newItemList);
-};
-
-const onClickDelete = id => {
-        const newItemList = items.filter(item => item.id !== id);
-        setItems(newItemList);
-};
-
-const onClickAdd = value => {
-	const newItemList = [
-		...items,
-		{
-			value,
-			isDone: false,
-			id: count + 1
-		}
-	];
-
-	setItems(newItemList);
-	setCount(count => count + 1);
+import styles from './App.module.css';
 
-}
+const App = () => (<Router>
+  <div className={styles.wrap}>
+    <MenuList className={styles.sidebar}>
+      <Link to='/' className={styles.link}>
+        <MenuItem>Обо мне</MenuItem></Link>
+      <Link to='/todo' className={styles.link}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 10.](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)